Glossary of Terms.md

```diff
@@ -83,18 +83,30 @@ A [**payload**](#def-payload) containing one specific [HTTP Response](http://www
 ### Resource
 An API [**resource**](http://www.w3.org/TR/di-gloss/#def-resource) specified by its *URI*. It can also refer to a [**set of resources**](#def-resource) matching one [**URI template**](#def-uri-template).
 
+<a name="def-resource-group"></a>
+### Resource Group
+An arbitrary grouping of one or more [**Resource Sets**](#def-resource-set).
+
 <a name="def-resource-model"></a>
 ### Resource Model
 One [**manifestation of a resource**](http://www.w3.org/TR/di-gloss/#def-resource-manifestation) in the form of a [payload](#def-payload). A resource model is an example representation of its resource. Can be referenced later in the place of a [payload](#def-payload).
 
 <a name="def-resource-set"></a>
 ### Resource Set
-A set of API [**resources**](http://www.w3.org/TR/di-gloss/#def-resource) its *URI* matches one specific  [**URI template**](#def-uri-template).
+A set of API [**resources**](http://www.w3.org/TR/di-gloss/#def-resource) with *URI*, which matches one specific [**URI template**](#def-uri-template).
```

Comment:
I suggest something along these lines: "A set of all resources with URIs matching one specific URI template"